app/controllers/ems_common.rb

```diff
@@ -780,9 +780,7 @@ def list_google_regions
   def retrieve_openstack_infra_providers
     openstack_infra_providers = []
     ManageIQ::Providers::Openstack::Provider.all.each do |provider|
-      openstack_infra_providers = [
-        [provider[:name], provider[:id]]
-      ]
+      openstack_infra_providers << [provider[:name], provider[:id]]
```

Comment:
@martinpovolny well pluck me, that is awesome method I didn't know about :-D thank youu